website/src/components/SequenceDetailsPage/SequencesContainer.tsx

```diff
@@ -154,15 +205,47 @@ const AlignmentSequenceTabs: FC<NucleotideSequenceTabsProps> = ({ nucleotideSegm
             {nucleotideSegmentNames.map((segmentName) => (
                 <BoxWithTabsTab
                     key={segmentName}
-                    isActive={isAlignedSequence(sequenceType) && segmentName === sequenceType.name}
-                    onClick={() => setType(alignedSequenceSegment(segmentName))}
+                    isActive={isActive && isAlignedSequence(sequenceType) && segmentName === sequenceType.name}
+                    onClick={() => {
+                        setType(alignedSequenceSegment(segmentName));
+                        setActiveTab('aligned');
+                    }}
                     label={`${segmentName} (aligned)`}
                 />
             ))}
         </>
     );
 };
 
+type GeneDropdownProps = {
+    genes: string[];
+    sequenceType: SequenceType;
+    setType: Dispatch<SetStateAction<SequenceType>>;
+};
+
+const GeneDropdown: FC<GeneDropdownProps> = ({ genes, sequenceType, setType }) => {
+    const selectedGene = isGeneSequence(sequenceType.name, sequenceType) ? sequenceType.name : '';
+
+    return (
+        <div className='mb-4'>
+            <select
+                className='select select-bordered w-full max-w-xs'
+                value={selectedGene}
+                onChange={(e) => setType(geneSequence(e.target.value))}
+            >
+                <option value='' disabled>
+                    Select a protein
```

Comment:
I think that we should avoid using the term "protein" here as not necessarily each AA seqeunce represents a different protein? Everywhere else, I believe that we have also rather just used "amino acid sequence" as term.

```suggestion
                    Select a sequence
```

Reply:
Hmm - you mean in that one protein can have multiple chains? In code we call these identifiers `genes` and I'd be OK with that here. I find `Select a sequence` a bit unexpected - here I'd expect the drop downs to be accessions. Would `Select a gene` work for you?

Reply:
For now, I think it's fine. When we look at more [schema designs](https://loculus.org/for-administrators/schema-designs/#multiple-references-for-an-organism) such as the multiple references for an organism approach, different AA sequences could be from the same gene aligned to different references.

Reply:
Sounds good to discuss then (I think in such a case I'd favour an approach where the UX involves selecting a reference and a gene(